src/style.css

```
@@ -1 +1,69 @@
 /* Styles go here */
+* {
+  margin: 0;
+}
+
+header {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+}
+
+.image {
+  display: flex;
+  padding-left: 50px;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  flex-shrink: 0;
+  margin: auto;
+}
+
+.nav-list {
+  list-style: none;
+  display: flex;
+  padding: 0;
+  flex-shrink: 0;
+}
+
+.nav__item {
+  padding-top: 20px;
+}
+
+.link {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  color: #000;
+  text-decoration: none;
+  text-transform: uppercase;
+  padding-top: 20px;
+  padding-right: 10px;
+  padding-left: 10px;
+  padding-bottom: 20px;
+  justify-content: center;
+}
+
+.link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+.is-active:after {
+  content: " ";
+  width: 37px;
+  height: 4px;
+  display: block;
+  position: absolute;
+  background-color: #00acdc;
+  border-radius: 8px;
+  margin-top: 18px;
+  margin-left: 10px;
+  margin-right: 10px;
+}
```

```suggestion
.is-active:after {
  content: " ";
  width: 100%;
  height: 4px;
  display: block;
  position: absolute;
  background-color: #00acdc;
  border-radius: 8px;
bottom: 0
}
```
you need to add relative position to the parent element!
use this site for more info)
https://css-tricks.com/absolute-positioning-inside-relative-positioning/